### PR TITLE
Change localization branch to release/8.0.1xx temporarily

### PR DESCRIPTION
**Pull request by carlossanlop**
Similar to the change we did in dotnet/runtime (https://github.com/dotnet/runtime/pull/90812), we need to temporarily switch the OneLocBuild MirrorBranch to point to the release/8.0.1xx branch to ensure all localization changes go there until we ship. This change will be reverted after we ship, so that we point to main again. We need to backport this to the [release/8.0.1xx](https://github.com/dotnet/roslyn-analyzers/tree/release/8.0.1xx) branch as well (like in https://github.com/dotnet/runtime/pull/90813).

@mavasani @sharwell @cristianosuzuki77 @buyaa-n @ericstj @ViktorHofer

### CODE REVIEW

**Comment by ViktorHofer** [comment]
I don't think that the backport GH is enabled in this repo.

**Review by ViktorHofer — APPROVED** [review]

**Review by buyaa-n — APPROVED** [review]
The changes make sense to me, but I don't know how the localizations was working in the branches before, I would wait for approval from @mavasani or @sharwell

**Pull request merged**
